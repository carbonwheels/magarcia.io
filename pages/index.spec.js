import React from 'react';
import renderer from 'react-test-renderer';
import Index from './index';

it('renders correctly', () => {
  const post = {
    id: 'id',
    slug: 'slug',
    title: 'title',
    content: 'content of the file',
    metadata: {
      url: 'http://example.com',
      intro: 'this is the intro',
      date: '2019-13-01',
      layout: 'post',
      tags: ['tag1', 'tag2'],
      readingTime: '5 mins'
    }
  };
  const props = { results: [post], total: 1, page: 1 };
  const tree = renderer.create(<Index {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
