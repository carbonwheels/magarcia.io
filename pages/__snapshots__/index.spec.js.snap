// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <ul
    className="list"
  >
    <li
      className="li"
    >
      <a
        className="titleLink"
        href="http://example.com"
        onClick={[Function]}
      >
        <h2>
          title
        </h2>
      </a>
      <div
        className="summary"
      >
        <div
          className="postContent"
        >
          <p>
            this is the intro
          </p>
        </div>
      </div>
      <div
        className="postMetaInline small"
      >
        <time
          dateTime={null}
        >
          Invalid date
        </time>
        <span
          className="middotDivider"
        />
        <span
          className="readingTime"
          title="5 mins"
        />
      </div>
    </li>
  </ul>,
  <div>
    <a
      className="page active"
    >
      1
    </a>
  </div>,
]
`;
